tagging-status/testfiles-partial/asmeconf/asmeconf-01-template-BAD.pdftex.struct.xml

```diff
@@ -2487,15 +2488,15 @@
          >
         <?MarkedContent page="5" ?>LATEX
        </Span>
-       <?MarkedContent page="5" ?>3 team added support for PDF/A to the
+       <?MarkedContent page="5" ?>3 team added support for PDF/A tothe
        <Span xmlns="http://iso.org/pdf2/ssn"
           id="ID.0572"
           actualtext="LaTeX"
           phoneme="[ˈlaːtɛx]"
          >
         <?MarkedContent page="5" ?>LATEX
        </Span>
-       <?MarkedContent page="5" ?>kernel, using the command \DocumentMetadata{..}.This approach works with both pdfLATEX and LuaLATEX. Notethat accessible conformance (a) is still under development by the
+       <?MarkedContent page="5" ?>kernel, using the command\DocumentMetadata{..}.This approach works with both pdfLATEX and LuaLATEX. Notethat accessible conformance (a) is still under development by the
```

Comment:
this is a BAD file so we'll come back to it at some point but this seems to be a regression, we lost the space here?

Reply:
yes sorry about that I noticed later diff chunks had spaces coming and going due to line changes, that first one just caught my eye as it was just losing a space. But anyway it is what it is, we can't change pdftex output in this cycle.